### PR TITLE
Fix issue that PnorUtils.pm places new layout file inside of buildroot

### PR DESCRIPTION
**Pull request by LF-Luis**
PnorUtils.pm’s loadPnorLayout now accepts an optional argument specifying where to place the new layout file. If argument is not given, it will be placed in the directory where the script is running.

Signed-off-by: Luis Fernandez <Luis.Fernandez@ibm.com>

### CODE REVIEW

**Comment by stewartsmith** [comment]
are these patches headed upstream? is there a PR to track them?

File: openpower/package/hostboot/0001-loadPnorLayout-accepts-optional-parameter-to-place.patch [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
++
+     my $filename = basename($i_pnorFile, ".xml");
+-    $filename = "${filename}WithOffsets.xml";
++
++    if (defined $i_outputLayoutLocation)
++    {
++        $filename = "${i_outputLayoutLocation}/${filename}WithOffsets.xml";
++    }
++    else {
++        # If argument $i_outputLayoutLocation is not supplied, the new layout
```
Comment (artemsen) [inline openpower/package/hostboot/0001-loadPnorLayout-accepts-optional-parameter-to-place.patch]: I think it is better to do not create the file at all, if output location is not specified.

**Comment by lpfluisfer** [comment]
> are these patches headed upstream? is there a PR to track them?

@stewartsmith I placed both patches here in an op-build commit to test in CI and get some feedback. Ultimately I'll make separate commits to the Hostboot and PNOR repo.

File: openpower/package/hostboot/0001-loadPnorLayout-accepts-optional-parameter-to-place.patch (hunk 1 above)
Comment (lpfluisfer) [reply openpower/package/hostboot/0001-loadPnorLayout-accepts-optional-parameter-to-place.patch]: Agree, made changes to reflect that.

**Comment by lpfluisfer** [comment]
The PNOR patch as a PR:
https://github.com/open-power/pnor/pull/130

**Pull request closed without merging**